lib/skate_web/controllers/detour_missed_stops_controller.ex

```diff
@@ -0,0 +1,47 @@
+defmodule SkateWeb.DetourMissedStopsController do
+  use SkateWeb, :controller
+
+  alias Skate.Detours.MissedStops
+  alias Util.Location
+
+  @spec missed_stops(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def missed_stops(conn, %{
+        "route_pattern_id" => route_pattern_id,
+        "connection_start" => connection_start,
+        "connection_end" => connection_end
+      }) do
+    route_pattern_fn =
+      Application.get_env(:skate_web, :route_pattern_fn, &Schedule.route_pattern/1)
+
+    shape_with_stops_fn =
+      Application.get_env(
+        :skate_web,
+        :shape_with_stops_fn,
+        &Schedule.shape_with_stops_for_trip/1
+      )
+
+    missed_stops_fn =
+      Application.get_env(:skate_web, :missed_stops_fn, &MissedStops.missed_stops/1)
```

Comment:
nit/suggestion(non-blocking): I have a little trouble focusing on the content of the function itself when having to also try and mentally ignore these setup env calls.
How would you feel about doing something similar to what I did in the `auth_controller` and instead define these as private functions?
https://github.com/mbta/skate/blob/a3c3e2495de9d27ee618dcb0db9359eda623af04/lib/skate_web/controllers/auth_controller.ex#L85-L90

I could also see `Mox` being a solution here? But I can't say I really understand it well enough yet to know how it'd be used.

Reply:
What do you think of 28f1c160?

Reply:
perfect! Thank you!